apps/create-fuels-counter-guide/src/app/layout.tsx

```diff
@@ -17,6 +17,7 @@ import {
 import { NODE_URL } from "@/lib";
 import { ActiveWalletProvider } from "@/hooks/useActiveWallet";
 
+// react-query is a peer dependency of @fuels/react, so we set it up here. See https://docs.fuel.network/docs/wallet/dev/getting-started/#installation-1
```

Comment:
Can we keep the comments line width within 80 chars to avoid horizontal scrolling and facilitate the reading? I'm commenting here, but the thought is valid to all comments.

Reply:
Good suggestion @arboleya, should be ok now

Reply:
I have no preference, but is there a difference between these two?

I wonder if the latter is more geared towards [type] docs. cc @danielbate 

We should stick to one.

```ts
/*
  Line 1
  Line 2 
*/
```

```ts
/**
  * Line 3
  * Line 4 
  */
```


Either way, and this is me being pedantic, but in the second case, there are a few misplaced whitespaces.

It makes the indentation a bit weird, although it's probably irrelevant:

```ts
/**
* Line 3
* Line r
*/
```

Reply:
My preference would be the latter as it's a common default keybinding for IDEs. If there's an indentation issue we may be able to resolve it with prettier.

`/*` to me indicates the comment is on one line.
`/**` is multiline.

Reply:
AFAICT, it is like this:

 - [JS Docs](https://jsdoc.app/about-getting-started)
 - [Single/multiline comments](https://www.w3schools.com/js/js_comments.asp)

```ts
// single line
const some = "thing";

/*
  multi
  line
*/
const something = "else";

/**
  * Use this var to configure your language,
  * according to ABC and XYZ.
  */
const language = "EN";
```

My concern was that using the latter `/**` for Docs could be rendered as documentation by Typedocs.

It may be irrelevant to the template app, but it could be different for the rest of the codebase.

Reply:
It doesn't seem to be interfering with typedoc, so I guess I'll leave it as it is